src/vs/workbench/contrib/inlineChat/browser/inlineChatStrategies.ts

```diff
@@ -325,6 +340,42 @@ export class LiveStrategy extends EditModeStrategy {
 		this._widget.updateStatus(message);
 	}
 
+	private _findEditsMaxLineNumber(response: EditResponse): number | void {
+		const edits = response.localEdits;
+		if (edits.length) {
+			let editsMaxLineNumber = 0;
+			for (const edit of edits) {
+				const editStartLine = edit.range.startLineNumber;
+				const editNumberOfLines = (edit.text.match(/\n/g) || []).length;
+				const endLine = editStartLine + editNumberOfLines - 1;
+				if (endLine > editsMaxLineNumber) {
+					editsMaxLineNumber = endLine;
+				}
+			}
+			return editsMaxLineNumber;
+		}
+	}
+
+	override getWidgetPosition(initialRender: boolean, _range: Range, response: SessionResponse): Position | undefined {
+		const viewModel = this._editor._getViewModel();
+		assertType(viewModel);
+		if (initialRender) {
+			this._initialPosition = viewModel.getPrimaryCursorState().viewState.position;
+			return this._initialPosition;
+		} else {
+			if (response instanceof EditResponse) {
+				const editsMaxLineNumber = this._findEditsMaxLineNumber(response);
```

Comment:
Instead of `_findEditsMaxLineNumber` you should be using `_session.lastTextModelChanges` - it describes the edits that have been applied so far and allows you to retrieve the affected line numbers